src/utils/displayFunctions.ts

```diff
@@ -64,11 +64,18 @@ export const makeDisplayFunctions = (brandToInfo: Map<Brand, BrandInfo>) => {
   const displayBrandIcon = (brand?: Brand | null) =>
     getLogoForBrandPetname(getPetname(brand));
 
-  const displayPrice = (price: PriceQuote) => {
-    return '$' + stringifyValue(price.amountOut.value, AssetKind.NAT, 2);
+  const displayPrice = (price: PriceDescription) => {
+    const PRICE_BRAND_UNIT_AMOUNT = 1_000_000n;
+    const USD_BRAND_DECIMALS = 6;
```

Comment:
these constants should be module-scope and well documented as to why they're necessary.

Reply:
done